frontend/src/components/Navi/Navi.js

```diff
@@ -16,13 +23,25 @@ const NaviDiv = styled.div`
 	}
 	z-index: 98;
 	position: fixed;
-	background-color: rgba(227, 220, 202, 0.85);
+	background-color: rgb(226, 207, 195);
+	box-shadow: 0px 3px 21px 0px rgba(0, 0, 0, 0.75);
 `;
```

Comment:
Don't think we will be using box-shadow since Josh Knell mentioned not to use it.